lib/model/model_test.go

```diff
@@ -947,11 +947,13 @@ func TestIgnores(t *testing.T) {
 
 	db := db.OpenMemory()
 	m := NewModel(defaultConfig, protocol.LocalDeviceID, "device", "syncthing", "dev", db, nil)
-	m.AddFolder(defaultFolderConfig)
-	m.StartFolder("default")
 	m.ServeBackground()
+	m.cfg.SetFolder(defaultFolderConfig)
 	defer m.Stop()
 
+	// wait for goroutine wrapper.notifyListener to finish
+	time.Sleep(time.Duration(5) * time.Millisecond)
```

Comment:
There is no `ConfigChanged` and `ConfigSaved` is not emitted as the config isn't saved to disk. So I adopted your solution - thanks!